openapi/time-registration-bulk.yaml

```diff
@@ -0,0 +1,32 @@
+post:
+  tags:
+    - "Time registration"
+  operationId: "addBulkTimeRegistrationForUser"
+  description: "Registers several time registrations for a user"
+  requestBody:
+    description: "A JSON object containing a list of time registration information"
+    required: true
+    content:
+      application/json:
+        schema:
+          type: "array"
+          items:
+            $ref: "./schemas/TimeRegistrationRequest.yaml"
+  responses:
+    "201":
+      description: "Created"
+      content:
+        text/plain:
+          schema:
+            type: "string"
+          examples:
+            Example#1:
+              value: "Time registered successfully"
+    "400":
+      description: "Failed creating one or more registrations"
```

Comment:
Some details about transaction scopes. Is it "all or none" or are the ones that are OK created, while the problematic ones are ignored? (I think we should do the "all or none", but let's be explicit about it in the api documentation)